pkgs/development/haskell-modules/lib.nix

```diff
@@ -126,6 +126,10 @@ rec {
    */
   dontDistribute = drv: overrideCabal drv (drv: { hydraPlatforms = []; });
 
+  /* overridePlatforms changes the supported platforms for a package.
```

Comment:
Can you add an example and type signature like this: <https://github.com/tazjin/nixdoc#comment-format>?